README.md

```diff
@@ -178,6 +178,70 @@ You can find our official [Open Source Maintenance Policy](open_source_maintenan
   </tr>
 </table>
 
+## OperationCode Infrastructure
```

Comment:
Looks good. Would you please also remove:
OperationCode Bot
OperationCode Slash Commands